pyatlan/errors.py

```diff
@@ -615,6 +615,13 @@ class ErrorCode(Enum):
         "Check the details of the server's message to correct your request.",
         InvalidRequestError,
     )
+    UNABLE_TO_CREATE_CREDENTIAL = (
+        400,
+        "ATLAN-PYTHON-400-071",
+        "When test is set to False, credentials can only be created without a username and password.",
+        "Either switch test to True or remove the username and password.",
+        InvalidRequestError,
+    )
```

Comment:
How about simplifying this to:

```suggestion
    UNABLE_TO_CREATE_CREDENTIAL = (
        400,
        "ATLAN-PYTHON-400-071",
        "Credentials cannot include a `username` or `password` when `test` is `False`.",
        "Set `test` to `True` or remove `username` and `password`.",
        InvalidRequestError,
    )
```